src/main/java/rx/subjects/UnicastSubject.java

```diff
@@ -35,6 +35,7 @@
  * 
  * @param <T> the input and output value type
  */
+@Experimental
```

Comment:
It is tested through `window` operators and there is `BufferUntilSubscriberTest`.